packages/components/src/TooltipTrigger/TooltipTrigger.component.js

```diff
@@ -114,13 +113,19 @@ function TooltipTrigger(props) {
 			tooltipWidth = DEFAULT_OFFSET_X,
 		} = props;
 
-		if (!refContainer.current) {
+		if (!refContainer || !refContainer.current) {
 			return {
 				tooltipPlacement,
 			};
 		}
 
-		const dimensions = refContainer.current.getBoundingClientRect();
+		let dimensions;
+		try {
+			// eslint-disable-next-line react/no-find-dom-node
+			dimensions = ReactDOM.findDOMNode(refContainer.current).getBoundingClientRect();
```

Comment:
Just a question about this change, I don't understand why you need to find your ref in the domnode, since the ref is normally a pointer to the domnode ?

Reply:
Because actually ref is not always referencing a DOM node, but just a pointer like you said, so there is no way to access to its `getBoudingClientRect`method.
> https://github.com/facebook/react/issues/14357#issuecomment-447276140